cmd/cluster/validatepullsecret.go

```diff
@@ -94,6 +95,9 @@ func (o *validatePullSecretOptions) run() error {
 
 func (o *validatePullSecretOptions) getPullSecretFromCluster() (email string, err error, sentSL bool) {
 	if o.elevate {
+		if len(o.reason) == 0 {
+			return "", errors.New("please provide a reason when using elevate"), false
+		}
```

Comment:
Can we show a prompt something like
`Reason not provided. Continue? (y/n)`

Reply:
That would not be consistent with other commands that requires reason, and what would be the benefit of not providing the reason in that case ? We just need to provide the ref to the PD/OHSS ticket ;)